docs/UserGuide.md

```diff
@@ -4,339 +4,1044 @@
+|-------------------------|---------------------------------------------------------------|
+| Command Box             | You can type commands here and press `Enter` to execute them. |
+| Result Display Box      | Displays the result of executing a command.                   |
+| List View               | Displays a list of your internship applications.              |
+| Intern Application Card | Displays the fields related to your internship application.   |
+| Select View             | Displays more details related to your internship application. |
 
-1. Type the command in the Command Box and press `Enter` to execute it. e.g. typing **`help`** and pressing `Enter` will open the help window. Here are some example commands you can try:
+Now that you are familiar with the user interface of LetsGetHired, let's jump right into managing your 
+internship applications in this [**Tutorial**](#tutorial-managing-your-first-internship-application).
 
-    * `list` : Lists all internship applications.
+__________________________________________________________________________________________________________________
 
-    * `add n/Google c/Summer r/Full Stack Developer ` : Adds a new internship application entry to the internship list.
+<div style="page-break-after: always;"></div>
 
-    * `delete 1` : Deletes the first internship application entry shown in the current list.
+## **Tutorial - Managing Your First Internship Application**
 
-    * `edit 1 s/Accepted` : Updates the status of the first internship application entry shown in the current list as `Accepted`.
+In this tutorial, we will walk you through a typical workflow for using LetsGetHired to manage your internship applications. <br>
 
-1. Refer to the [Features](#features) below for details of each command.
-1. Follow the [Tutorial](#tutorial---managing-your-internship-applications) below to use LetsGetHired for your first internship application.
+### Step 1: Adding Your Internship Application
 
---------------------------------------------------------------------------------------------------------------------
+**Context:** You are starting your internship application journey, and you want to add a new internship application.
 
-## Tutorial - Managing Your Internship Applications
+Imagine that you have applied for a **Software Engineering Internship** at **Google** for the **Summer of 2024**.
 
-In this tutorial, we'll walk you through a typical workflow for using LetsGetHired to manage your internship applications. Let's imagine that you want to add an entry for a software engineering internship at Google for the summer of 2024.
+To **add your internship application**, go to the _Command Box_ and type the following command:
+```
+add n/Google r/Software Engineering c/Summer 2024
+```
+This command specifies the company (_Google_), the role (_Software Engineering_), and the cycle (_Summer 2024_) for the internship that you have applied for.
```

Comment:
Maybe we can tell the user the default status is pending here, or change the command to specify the status

Reply:
Maybe we can tell the user that the details of the added application are automatically shown in the select view as well

Reply:
> Maybe we can tell the user the default status is pending here, or change the command to specify the status

I didn't feel like overloading the user with information here. The default values are provided later on in the "Add" command section!